pkg/oam/util/helper_test.go

```diff
@@ -715,11 +716,37 @@ var _ = Describe("Test unstructured related helper utils", func() {
 				}},
 				exp: "simplerollouttraits.extend.oam.dev",
 			},
+			"trait": {
+				u: &unstructured.Unstructured{Object: map[string]interface{}{
+					"apiVersion": "extend.oam.dev/v1alpha2",
+					"kind":       "SimpleRolloutTrait",
+					"metadata": map[string]interface{}{
+						"labels": map[string]interface{}{
+							oam.TraitTypeLabel: "rollout",
+						},
+					},
+				}},
+				typeLabel: oam.TraitTypeLabel,
+				exp:       "rollout",
+			},
+			"workload": {
+				u: &unstructured.Unstructured{Object: map[string]interface{}{
+					"apiVersion": "apps/v1",
+					"kind":       "Deployment",
+					"metadata": map[string]interface{}{
+						"labels": map[string]interface{}{
```

Comment:
it will be good to add a name here that is different from the label so we know which is the truth

Reply:
I don't know what do you mean here? Which name?